examples/basic/README.md

```diff
@@ -2,11 +2,13 @@
 
 ## Building
 
+You can build this example from the root of the `purescript-react-dnd-basic` project:
+
 ```sh
 npm install
-make all
+npm run example:basic
 ```
 
-This will compile the PureScript source files, bundle them, and use Browserify to combine PureScript and NPM sources into a single bundle.
+This will compile the PureScript source files, bundle them, and combine PureScript and NPM sources into a single bundle.
```

Comment:
Changed this to be a bit more like the Halogen examples: https://github.com/purescript-halogen/purescript-halogen/tree/master/examples/basic (removed the Makefile and `package.json` from this directory in favor of running `spago` from root directory and adding NPM deps to top-level `devDependencies`)